python/adelphi/README.md

```diff
@@ -0,0 +1,107 @@
+# Adelphi tooling
+
+## Commands
+
+### export-cql
+This command extracts schemas for the specified keyspaces from a Cassandra instance and then displays the CQL commands necesary to generate them to standard out.  You can optionally specify an output directory, in which case the CQL commands are written to files within that directory, one file for each keyspace.
```

Comment:
If we change the output structure so that we get a directory and a series of files plus the metadata file we'll need to remember to come back here and update this.

Reply:
Based on the help dump below, I think this command is optional, and if it's left out you get all keyspaces by default - can we include a note about that here?

Reply:
Actually, I guess looking at the other blocks here that default behavior is more for the --keyspaces option than it is the individual commands, so no need to repeat it over and over again.  But, it makes me wonder if we shouldn't pull the options section up higher and then expand on the detailed commands afterwards maybe?

Reply:
typo: necesary

Reply:
Your comments did highlight a couple things that are missing here @jdonenine :

* If keyspaces aren't specified all keyspaces are selected, at least initially
* System keyspaces are removed from the list whether you explicitly specify keyspaces or leave them out and select everything

I'll add comments about both those points to the docs.

Reply:
I wound up adding a section to the "Options" block describing how keyspace selection works.  I think it covers all the high points we discussed here.  Can you guys double-check and confirm that everything that needs to be there is there and that it's clear?

Reply:
I wound up adding a section to the "Options" block describing how keyspace selection works.  I think it covers all the high points we discussed here.  Can you guys double-check and confirm that everything that needs to be there is there and that it's clear?